src/pages/Octopath-Traveler-II/Full-Game/Any-Hikari-DPS.mdx

```diff
@@ -949,7 +950,7 @@ Guide Story Guy.
 #### Kaldena 100000HP 8S (Spear Staff Ice Wind) Collect $1200
 - Hikari: Sixfold Strike
 - Agnea: Crit Scope
-- Throne: Abating Orb
+- Throne: Abating Orb x2
```

Comment:
I'm actually not sure about this one. I saw you boosting in your run while I was struggling due to an unrelated problem.

Reply:
I'm going to leave orb as x1, just so people don't think x2 is required. Boosting just cuts 1 directional input.